cime_config/machines/config_machines.xml

```diff
@@ -620,10 +618,8 @@
       </modules>
 
       <modules>
-        <command name="rm">git</command>
-        <command name="load">git</command>
```

Comment:
@ndkeen - did you mean to remove the git module?

Reply:
Yea -- looks like the removed the module. We never needed a specific version, I just thought it was safer to do so. 
```
cori09% module load git
ModuleCmd_Load.c(244):ERROR:105: Unable to locate a modulefile for 'git'
cori09% which git
/usr/bin/git
cori09% git --version
git version 2.26.2
```

Reply:
It does cause problems not to have it -- one of our users reported this:
````
----- Starting case_setup -----
Configuring E3SM to use the COSP simulator
Successfully cleaned env_mach_specific.xml
ERROR: module command /opt/modules/default/bin/modulecmd python load git failed with message:
ModuleCmd_Load.c(244):ERROR:105: Unable to locate a modulefile for 'git'
````

Reply:
That error message was from an e3sm runscript that previously worked

Reply:
Right, if there isn't a git module, then loading it will cause an error in CIME (unless we ignore module errors, which might be dangerous) -- therefore, only thing to do is NOT try to load the module.

Reply:
The runscript isn't trying to load the git module -- so CIME must be trying to somewhere under the covers. Is there a reason not to load the git module?

Reply:
Yes.  In fact, there is no git module. NERSC removed it.  This change will correct the problem.

Reply:
Thanks @ndkeen -- I think I understand what's going on now. I appreciate all your help